volume/volume.go

```diff
@@ -104,8 +110,9 @@ type MountPoint struct {
 	CopyData bool `json:"-"`
 	// ID is the opaque ID used to pass to the volume driver.
 	// This should be set by calls to `Mount` and unset by calls to `Unmount`
-	ID   string `json:",omitempty"`
-	Spec mounttypes.Mount
+	ID    string `json:",omitempty"`
+	Tmpfs *Tmpfs `json:"-"`
```

Comment:
Put differently, let's not cut `docker run` over to using the mounts for tmpfs until there is parity.

I think we need to understand the use case of `uid/gid` and `nr_blocks` to better support these across operating systems. Right now, these are extremely non-portable.